{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/core/models.py

```diff
@@ -1,9 +0,0 @@
-from django.db import models
```

Comment:
The only issue that I found was in regards to this file. Don't forget to delete it if it doesn't include any code.

Reply:
ok thanks!